rosbag2_cpp/include/rosbag2_cpp/writers/sequential_writer.hpp

```diff
@@ -77,6 +77,12 @@ class ROSBAG2_CPP_PUBLIC SequentialWriter
    *
    * \param storage_options Options to configure the storage
    * \param converter_options options to define in which format incoming messages are stored
+   * \throws runtime_error if 
+   * database directory already exists,
+   * failed to create database directory,
+   * no storage could be initialized,
+   * invalid bag splitting size given,
+   * max cache size less or equal 0 when snapshot mode is enabled.
```

Comment:
I am not sure if we want to list the all possible exception here, but at least indent is needed as below.

```suggestion
   * \throws runtime_error if database directory already exists,
   *   failed to create database directory, no storage could be initialized,
   *   invalid bag splitting size given,
   *   max cache size less or equal 0 when snapshot mode is enabled.
```